linux/cuda_info.bsh

```diff
@@ -976,6 +976,56 @@ function tcnn_cuda_architectures()
 }
 
 
+#**
+# .. function:: nvcc_gencodes
+#
+# :Arguments: * ``$1`` - delimited string of cuda architectures
+# :Output: *stdout* - string containing nvcc gencode statements
+#
+# Create nvcc gencode statements from input array of cuda capabilities.
+#
+# .. rubric:: Example
+#
+# For example, an input of ``"75 86+PTX"`` would produce the output
+#
+# .. code-block:: bash
+#
+#   -gencode=arch=compute_75,code=sm_75 -gencode=arch=compute_86,code=compute_86
+#
+#**
+function nvcc_gencodes()
+{
+  # input as space delimited string
+  local input="${1//[,;]/ }"
+
+  # remove any periods from cuda architectures (e.g., 7.5->75)
+  input="${input//./}"
+
+  # input as array
+  local capabilities=( ${input} )
```

Comment:
```suggestion
  local capabilities=( ${input} ) # noquotes
```

Because it's so rare this is what we actually want to do, add this comment to let the formatter know it's ok, and future us